docs/source/installation.rst

```diff
@@ -23,19 +23,19 @@ Install the package from the ``conda-forge`` channel:
 
     conda install dask-sql -c conda-forge
 
-Experimental GPU support
+GPU support
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
 - GPU support is currently tied to the `RAPIDS <https://rapids.ai/>`_  libraries.
-- It is in development and generally requires the latest `cuDF/Dask-cuDF <https://docs.rapids.ai/api/cudf/legacy/10min.html>`_ nightlies.
-- It is experimental, so users should expect some bugs or undefined behavior.
+- It generally requires the latest `cuDF/Dask-cuDF <https://docs.rapids.ai/api/cudf/legacy/10min.html>`_ nightlies.
```

Comment:
We can point to the nightly link here: https://docs.rapids.ai/api/cudf/nightly/user_guide/10min.html